gadget.Dockerfile

```diff
@@ -33,10 +33,10 @@ FROM docker.io/kinvolk/traceloop:202006050210553a5730 as traceloop
 # Main gadget image
 
 # BCC built from:
-# https://github.com/kinvolk/bcc/commit/321c9c979889abce48d0844b3d539ec9a01e6f3c
+# https://github.com/kinvolk/bcc/commit/c5ad55e0067b4d87e503f80367066c5ae1d7acac
```

Comment:
> the comment should be updated to say we use the branch "gadget"

I meant this comment in addition to the commitmsg:
 
```suggestion
# BCC built from the gadget branch at:
#
# https://github.com/kinvolk/bcc/commit/c5ad55e0067b4d87e503f80367066c5ae1d7acac
```

Reply:
Ah, I thought you were referring to the commit message. I updated the comment in the docker file as well.